### DISCUSSION TITLE
File upload Request

### DISCUSSION THREAD

**Discussion by plumped** [photoprism/photoprism#2733] [Ideas]
As i know it from Synology DSM and Synology Photos, it would be very nice to have a function to request file upload to a specifig album.

1. Share a file upload request link
2. Colleagues an Family upload pictures directly to the album for lets say after vacation or something.

This is the only missing function so far to combine the Taken pictures into one album (mine)

Something planed in this direction?

**Comment by graciousgrey** [comment]
Many improvements are planned for sharing :)

https://github.com/photoprism/photoprism/issues/557

https://github.com/photoprism/photoprism/issues?q=is%3Aissue+is%3Aopen+sharing+in%3Atitle
